benchmarks/query_benchmarks/query_complex_filter/benchmark.py

```diff
@@ -0,0 +1,11 @@
+from ...utils import bench_setup
+from .models import Book
+
+
+class QueryCmplxFilter:
+    def setup(self):
+        bench_setup(migrate=True)
+
+    def time_query_complex_filter(self):
+        for i in range(0, 10):
+            Book.objects.complex_filter({"pk": f"{i}"})
```

Comment:
```suggestion
            Book.objects.complex_filter({"pk": "1"})
Book.objects.complex_filter({"pk": "2"})
...
```

Can we write this out over 10 lines?